client/asset/btc/btc.go

```diff
@@ -1184,13 +1194,24 @@ func newUnconnectedWallet(cfg *BTCCloneCFG, walletCfg *WalletConfig) (*baseWalle
 	}
 	w.cfgV.Store(baseCfg)
 
-	if w.estimateFee == nil {
-		w.estimateFee = w.feeRate
+	// Default to the BTC RPC estimator (see LTC). Consumers can use
+	// NoLocalFeeRate or a similar dummy function to power feeRate() requests
+	// with only an external fee rate source available. Otherwise, all method
+	// calls must provide a rate or accept the configured fallback.
+	if w.localFeeRate == nil {
+		w.localFeeRate = rpcFeeRate
 	}
 
 	return w, nil
 }
 
+// NoLocalFeeRate is a dummy function for BTCCloneCFG.FeeEstimator for a wallet
+// instance that cannot support a local fee rate estimate but has an external
+// fee rate source.
+func NoLocalFeeRate() (uint64, error) {
+	return 0, errors.New("no local fee rate estimate possible")
```

Comment:
Not used, but an idea to illustrate what these comments in the constructor are describing.